postgres/schema.sql

```diff
@@ -0,0 +1,87 @@
+CREATE EXTENSION btree_gist;
```

Comment:
need some note that all this is from layersdb really

Reply:
what exactly to do?